hack/preview-template.env

```diff
@@ -13,16 +21,9 @@ export MY_GITHUB_TOKEN=
 # Optional
 
 ## Cluster and KCP config
-### Set cluster kubeconfig so it's not needed to pass it as parameter
-### example: $HOME/.kube/config
-export CLUSTER_KUBECONFIG=
-### KCP cluster kubeconfig so it's not needed to pass it as parameter
-### This file is managed by preview script
-### example: $HOME/.kube/preview-kcp
-export KCP_KUBECONFIG=
-### Path to CPS kubeconfig which will be copied when using preview-cps mode
-### example: $HOME/.kube/cps
-export CPS_KUBECONFIG=
+### KCP Workspace to deploy, default "~"
+### example: root:users:test
```

Comment:
We should not create any workspace under `root:users` - there should be present only the buckets 
```suggestion
### example: root:test
```